libphal/phal_dump.C

```diff
@@ -11,14 +11,14 @@ extern "C" {
 #include "phal_exception.H"
```

Comment:
commit message replace ipl: phal --> libphal 

Add additional description and tested details.

Reply:
done